src/it/scala/org/zalando/spearheads/innkeeper/routes/AcceptanceSpecsHelper.scala

```diff
@@ -20,7 +20,7 @@ import scala.language.implicitConversions
  */
 object AcceptanceSpecsHelper extends ScalaFutures with Matchers {
 
-  private val baseUri = "http://localhost:8080"
+  val baseUri = "http://localhost:8080"
```

Comment:
To be able to use it in tests: ```val response = doGet(s"$baseUri/hosts", READ_TOKEN)```. This code is clean enough and having separate method that will wrap this logic sounds like and overkill.